riposte-core/src/test/java/com/nike/riposte/server/testutils/TestUtil.java

```diff
@@ -0,0 +1,56 @@
+package com.nike.riposte.server.testutils;
+
+import com.nike.riposte.server.channelpipeline.ChannelAttributes;
+import com.nike.riposte.server.http.HttpProcessingState;
+import com.nike.wingtips.Span;
+import com.nike.wingtips.Tracer;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.util.Attribute;
+
+import java.util.Deque;
+import java.util.HashMap;
+import java.util.LinkedList;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class TestUtil {
+
+    public static ChannelHandlerContext mockChannelHandlerContext() {
+        ChannelHandlerContext mockContext = mock(ChannelHandlerContext.class);
```

Comment:
These methods should return a simple data structure that has all the mocks in it, so that users can easily add more mocking setup.

Reply:
I've refactored the methods to be more clear and return the mocks to allow further setup / config.